### PR TITLE
Lighthouse improvements

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

* Adds Github Action that runs lighthouse on deploy and updates the previous comment with the new report if there is one, otherwise, it creates a new comment.
* Some improvements:
  - adds a title to the theme switcher
  - removes unneeded fonts
  - fixes canonical_url in review apps
  - fixes canonical urls in API reference pages
  - adds `rel=noopenener` to external links
  - remove duplicates `id`s
  - updates badges so they are volta@a11y compliant
  - defers and preloads js/css
  - updates volta to a11y
  - adds `Cache-Control` to static files

### CODE REVIEW

File: lib/nexmo_developer/Gemfile [hunk 1]
```diff
@@ -129,7 +129,7 @@ gem 'lograge'
 gem 'countries'
 gem 'country_select', '~> 4.0'
 
-gem 'nexmo-oas-renderer', '~> 2.4.1', require: false
+gem 'nexmo-oas-renderer', github: 'nexmo/nexmo-oas-renderer', branch: 'add-title-to-switcher', require: false
```
Comment (bencgreenberg) [inline lib/nexmo_developer/Gemfile:132]: I think the PR on this branch was merged and released?

File: lib/nexmo_developer/app/webpacker/javascript/volta/volta.js [hunk 2]
```diff
@@ -310,6 +310,11 @@ Volta.accordion = function () {
 				this.trigger.addEventListener('click', function(){
 					self.toggle();
 				});
+				this.trigger.addEventListener('keydown', function(event) {
```
Comment (bencgreenberg) [inline lib/nexmo_developer/app/webpacker/javascript/volta/volta.js:313]: 💙 This is great!

File: lib/nexmo_developer/Gemfile (hunk 1 above)
Comment (fabianrbz) [reply lib/nexmo_developer/Gemfile:132]: I'll check whether this was released or not and update it accordingly 👍

**Review by bencgreenberg — APPROVED** [review]
These are all great changes, and I am ✅ , with the exception of just making sure that the dependency is not running off of the branch as mentioned earlier, as I believe that has been merged and released.

**Comment by fabianrbz** [comment]
@bencgreenberg @mheap rebased!

**Comment by mheap** [comment]
Previous scores:

ADP Homepage

ADP Markdown

ADP API Page

**Pull request merged**